client/libproject/devstart.rb

```diff
@@ -0,0 +1,17 @@
+require "optparse"
+# How can we tell what symbols come out of this import?
```

Comment:
You can't and, you're right, it sucks. So one uses a naming convention instead: `require "foo"` gives you `Foo`.